contrib/docker/README.md

```diff
@@ -12,7 +12,7 @@ use that server.
 Build the docker image with the `docker build` command from the root of the synapse repository.
 
 ```
-docker build -t matrixdotorg/synapse:v0.22.1 .
+docker build -t matrixdotorg/synapse .
```

Comment:
After the change in the compose file (the `docker.io/` prefix), you should probably change it here too. If someone builds the image and then tries to run the compose file, docker-compose will still try to pull in the image from docker.io